server/master/src/test/java/org/apache/accumulo/master/replication/UnorderedWorkAssignerTest.java

```diff
@@ -24,15 +24,11 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
This is causing checkstyle to fail.  Your IDE might have done this for you but we don't use start imports.
```
src/test/java/org/apache/accumulo/master/replication/UnorderedWorkAssignerTest.java:[27] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - java.util.*.
```